packages/server/tests/localAcceptance.env

```diff
@@ -27,3 +27,4 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
+SDK_REQUEST_TIMEOUT=12000
```

Comment:
qq: why do we need this?

Reply:
The SDKClient timeout occurred in this CI job: https://github.com/hashgraph/hedera-json-rpc-relay/actions/runs/11142264400/job/30964850632. This is what clued me into looking for time outs on mainnet, so the solution is to update the logged warning to make finding these on mainnet easier, going forward, and to increase the SDKClient time out on both mainnet and CI.